index.html

```diff
@@ -14,5 +14,6 @@
 
 
 <script src="main.js"></script>
+<script src="data.js"></script>
```

Comment:
Load the data model first, it won't make a difference now but it can trip a bug later.